README.md

```diff
@@ -87,6 +87,34 @@ To test the QA-signed extension in Firefox, you must be running the Nightly rele
 
 > **Note:** this will cause production-signed extensions (such as those from addons.mozilla.org) to not load. To allow these, set `xpinstall.signatures.required` pref to `false`.
 
+## Testing Remote Settings changes
+
+The Rally core-addon uses the Firefox [remote settings server](https://remote-settings.readthedocs.io/en/latest/) to publish metadata about current studies.
+Changes may be tested locally, or on the dev and staging servers, before pushing live.
```

Comment:
What's needed for testing locally? Can any HTTP server work (by just serving the JSON file)?

Reply:
Hmm maybe... I was able to get it working for tests at least :) Honestly it's probably easiest to use the dev server I can link to https://remote-settings.readthedocs.io/en/latest/tutorial-dev-server.html and maybe https://remote-settings.readthedocs.io/en/latest/tutorial-local-server.html for setting up a local server.

Reply:
Although, as we discussed in other comment threads here, if we allow the `build-local-addon` feature to continue working then in general for folks building locally in dev mode it'd be easiest to use that method.

Using an actual RS server is probably more for people doing QA on builds, or testing something very specific to our RS integration.